crates/runtime/src/env/mod.rs

```diff
@@ -35,16 +37,12 @@ pub use rocksdb::{RocksAppStore, RocksTemplateStore};
 #[cfg(feature = "default-rocksdb")]
 pub use default::{DefaultRocksAppStore, DefaultRocksEnvTypes, DefaultRocksTemplateStore};
 
-/// Runtime traits
 mod traits;
 
-pub use traits::{
-    AppAddressCompute, AppStore, TemplateAddressCompute, TemplateHasher, TemplateStore,
-};
+pub use traits::{AddressLocator, AppStore, TemplateHasher, TemplateStore};
 
-/// Runtime types
-pub mod hash;
-use hash::TemplateHash;
+/// Represents an `Template` Hash.
+pub type TemplateHash = [u8; 32];
```

Comment:
In most cases I prefer to use a `newtype` since I can extend it with functionality.
However, in this case, it seems reasonable to use alias instead.

Reply:
I should have asked about this and decided together. Lesson learnt :)

Reply:
No worries. when I use `newtype` I usually add `#[repr(transparent)]`
But the code base is also using some aliases

It's not a black or white I think